snooty/rstspec.toml

```diff
@@ -1434,6 +1434,27 @@ type = {link = "https://mongodb.github.io/node-mongodb-native/3.6/api/%s"}
 [role."xamarin"]
 type = {link = "https://docs.microsoft.com/en-us/xamarin/%s"}
 
+[role."k8sdocs"]
+type = {link = "https://kubernetes.io/docs%s"}
+
+[role."gopkg"]
+type = {link = "https://godoc.org%s"}
+
+[role."osi"]
+type = {link = "https://opensource.org%s"}
+
+[role."osil"]
+type = {link = "https://opensource.org/licenses/%s"}
+
+[role."tldrl"]
+type = {link = "https://www.tldrlegal.com/l/%s"}
+
+[role."q-mdb"]
```

Comment:
Are these existing roles in legacy, or could we, say, call these e.g. `quay-mongodb`? I'd broadly much rather roles not be too acryonym'd

Reply:
These exist currently in the k8s operator source; if we decide to change it we'll need to update those as well. 🤷🏻‍♀️